### ISSUE TITLE
Modifier keys stuck when using `Input.is_key_pressed` check

### ISSUE THREAD

**Issue by hedin-hiervard** [godotengine/godot#29798]
**Godot version:**
3.1

**OS/device including version:**
MacOS 10.12.6 (I think this is highly relevant here).

**Issue description:**
I'm checking if some modifier keys are pressed using:
```
Input.is_key_pressed(KEY_SHIFT)
Input.is_key_pressed(KEY_META)
Input.is_key_pressed(KEY_CONTROL)
```

Sometimes the method returns `true` despite the fact the key is NOT pressed. I think it might be related to some internal state and not actually checking with the driver. It happens for example when quickly cmd-tabbing between the game and the editor. I often find myself in the position when the Command key (KEY_META) is stuck. Pressing and releasing it helps (obviously some state is cleared).

**Comment by DustinStuff** [comment]
Confirmed that this issue also exists on Windows. 

Easiest way to reproduce this is to add something like this to some scene:  
```
func _process(delta):
    print(Input.is_key_pressed(KEY_ALT))
```
Run the scene, then alt-tab out of it and check the output. Unsurprisingly, this bug also affects `Input.is_action_pressed()` checks.

**Comment by hedin-hiervard** [comment]
> Unsurprisingly, this bug also affects `Input.is_action_pressed()` checks.

No, `Input.is_action_pressed()` officially ignores modifiers, which makes it totally useless. I use a custom version of it, which actually cares about modifiers, but this bug makes it tough as well.

**Comment by KoBeWi** [comment]
If this is really about Alt-Tabbing, it was fixed in 3.2 (I tested and don't see the issue anymore). Comment if you can still reproduce in 3.2.2.
